.github/actions/pylint-runner/entrypoint.sh

```diff
@@ -2,6 +2,7 @@
 
 set -u
 
+pipenv requirements > requirements.txt
```

Comment:
This is nice - looks like `pipenv requirements` is a new feature as of a couple of months ago. It's great that the github runner can support it.